vendor/github.com/zmap/zcrypto/x509/README.md

```diff
@@ -0,0 +1,8 @@
+Originally based on the go/crypto/x509 standard library,
+this package has now diverged enough that it is no longer
+updated with direct correspondence to new go releases.
```

Comment:
This is vaguely concerning. What's the level of comfort with a non-stdlib crypto library?